br/pitr-troubleshoot.md

```diff
@@ -0,0 +1,84 @@
+```
+
+At this time, you need to confirm the cause of the failure and resolve it according to the error message. To restore the backup task after resolving the failure, you can use the following command:
+
+```shell
+br log resume --task-name=task1 --pd x.x.x.x:2379
+```
+
+After the backup task is restored, you can check the status using `br log status` again. The backup task continues when the task status becomes `NORMAL`.
+
+```shell
+● Total 1 Tasks.
+> #1 <
+              name: task1
+            status: ● NORMAL
+             start: 2022-07-25 13:49:02.868 +0000
+               end: 2090-11-18 14:07:45.624 +0000
+           storage: s3://tmp/br-log-backup0ef49055-5198-4be3-beab-d382a2189efb/Log
+       speed(est.): 15509.75 ops/s
+checkpoint[global]: 2022-07-25 14:46:50.118 +0000; gap=6m28s
+```
+
+> **Note:**
+>
+> Since the feature backs up multiple versions of data, when the task fails and the status becomes `ERROR`, the checkpoint data of the current task is set as a `safe point`, and the data of the `safe point` will not be garbage collected within 24 hours. So, when the task is restored, the backup task continues from the last checkpoint. If the task fails for more than 24 hours, the last checkpoint data has been garbage collected, and you can only execute the `br log stop` command to stop the task first and then start a new backup task.
+
+## What should I do if the error message `ErrBackupGCSafepointExceeded` is returned when executing the `br log resume` command to resume the suspended task?
+
+```shell
+Error: failed to check gc safePoint, checkpoint ts 433177834291200000: GC safepoint 433193092308795392 exceed TS 433177834291200000: [BR:Backup:ErrBackupGCSafepointExceeded]backup GC safepoint exceeded
+```
+
+After pausing the log backup task, the pausing task program set the current checkpoint as service safepoint automatically to prevent the MVCC data generated by the changelog from being garbage collected. The MVCC data can be saved in 24 hours. When more than 24 hours, the MVCC data of the checkpoint has been garbage collected, and the backup task is unable to resume.
```

Comment:
```suggestion
After you pause a log backup task, to prevent the MVCC data from being garbage collected, the pausing task program sets the current checkpoint as the service safepoint automatically. This ensures that the MVCC data generated within 24 hours can remain. If the MVCC data of the backup checkpoint has been generated for more than 24 hours, the data of the checkpoint will be garbage collected, and the backup task is unable to resume.
```